tour-of-go/exports/config/config.go

```diff
@@ -4,7 +4,7 @@ import "fmt"
 
 var GameName string = "FiFa 2023"
 
-func Namevex() {
+func FetchNamevet() {
```

Comment:
Correct the this. change it to `fetchNameVex()"

All new words after the first word in a function name must start with capital letter.


Please check all the places in your packages where you made this same mistake and correct them.

Reply:
corrected